### DISCUSSION TITLE
Error list

### DISCUSSION THREAD

**Discussion by shisever** [Anura3D/Anura3D_OpenSource#160] [Q&A]
Hi,

I was trying to install anura3D using VS 17.8.0 and oneAPI 2024.0.0. I recieved a following list of errors:
![2023-11-18_23-50-44](https://github.com/Anura3D/Anura3D_OpenSource/assets/114015063/a135436b-3d4f-40a7-9027-bdca32427573)
are there any recommendations or suggestions? I would be really grateful

**Comment by jacobosaa** [comment]
Hi shisever! There's a youtube video that is pretty helpful for installing the software, you can use that video in parallel with the tutorial pdf and it should work smoothly. Did you download the oneAPI with the HPC toolkit? it has a certain order to download.  https://www.youtube.com/watch?v=dg9TOtY9YE4&list=PL4fEqhlQ0lQpsQC_8_I5tLWGr7sjAw8fl&index=1&ab_channel=Anura3DSoftware

Hoping it helps you!
Jacobo

**Reply by shisever** [reply]
Hi Jacobo! I did indeed check both manual and youtube tutorial before and during installation process as well as microsoft and intel websites and recommendations. Sadly, not following the instructions is not the key to my problem. Even after trying to reinstall everything all over again, I only managed to get rid of 5 errors which still brings me to the list of 19

**Reply by murphy-john** [reply]
Hello @shisever, I also just started to get this error. My laptop crashed and I had to reload the programs and files. I think it has todo with the 2024 update to oneAPI conflicting with our code. We will look into it and get back to you. 

Which versions of Visual Studio and Intel oneAPI are you using?

**Comment by PetitPipi** [comment]
Did you solve the problem?  I encountered a very similar one, and then I found that line 55 in _WriteOutPut_GiD.for_ might be the problem. Maybe try commenting out this line; just add an exclamation mark '!' in front of the line like this:
! #pragma comment(lib, "gidpost.lib")

I hope this helps.

**Reply by shisever** [reply]
Hi! Thanks, I saw ur fork when I was installing anura, it was helpful but I still had some errors left. In my case, I switched from VS2022 to VS2017 and only then build process went with no errors and I finally got my executable. I guess it's still more compatible with older versions.

**Reply by Amitanandan** [reply]
hey thanks could you tell what the intel api tools you have used? and could you send the Executebale file ? I am unable to do it...

**Reply by murphy-john** [reply]
You need the oneAPI Base Toolkit and the oneAPI HPC toolkit.
